Dockerfile

```diff
@@ -1,3 +1,5 @@
 FROM cpclermont/lighthouse-ci-action:1.0.0
+RUN gem uninstall shopify-cli
+RUN gem install shopify-cli -N -n /usr/local/bin
```

Comment:
It seems that more recent versions of `shopify-cli` gem 2.x are installed to a different directory by default and require some changes to the PATH. Installing the executable to `/usr/local/bin` gets around this.